packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -335,7 +335,7 @@ describeNoCompat("Named root data stores", (getTestObjectProvider) => {
             assert.ok(await getRootDataStore(dataObject1, alias));
         });
 
-        itExpects("Creating a root data store with an existing alias as an id breaks the container", [
+        itExpects.skip("Creating a root data store with an existing alias as an id breaks the container", [
```

Comment:
Not sure if these failures still happen. https://github.com/microsoft/FluidFramework/issues/9371

Reply:
Found this failure in tinylicious: https://dev.azure.com/fluidframework/internal/_build/results?buildId=56170&view=ms.vss-test-web.build-test-results-tab&runId=1174698&resultId=104669&paneView=debug

Reply:
`3/4/2022, 6:50:49 PM`. The fix went in on `3/7/2022`

Reply:
Removed skip from this test